CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Unreleased
 
-- Fix: Add support to show-hide-content.js for form fields with more varied characters: ([PR 403](https://github.com/alphagov/govuk_frontend_toolkit/pull/403))
+- Breaking change: Add support for GA Universal in stageprompt.js
+
+  Also ensures that events triggered via GA Universal do not impact bounce-rate calculations,
```

Comment:
Does this need mentioning, as it's not a change from anything that's been released?

Reply:
Given we know of forks that do not include this, I think it's worth mentioning.